packages/datadog-plugin-fetch/test/integration-test/server.mjs

```diff
@@ -1,9 +1,6 @@
 import 'dd-trace/init.js'
-import getPort from 'get-port'
 
-const port = await getPort()
-
-global.fetch(`http://localhost:${port}/foo`)
+global.fetch('http://localhost:0/foo')
```

Comment:
If I am not mistaken, this will fall back to port 80? This should in my opinion not use 0 as port

Reply:
Updated with an explicit port. The port doesn't matter anyway, but I can see how `0` could raise eyebrows.